release-notes/5.0/5.0-supported-os.md

```diff
@@ -1,50 +1,84 @@
 
 ## Windows
 
-OS                            | Version                       | Architectures  | Notes
-------------------------------|-------------------------------|----------------|-----
-Windows Client                | 7 SP1+, 8.1                   | x64, x86       |
-Windows 10 Client             | Version 1607+                 | x64, x86, ARM64       |
-Nano Server                   | Version 1803+                 | x64, ARM32     |
-Windows Server                | 2012 R2+                  | x64, x86       |
+OS                                    | Version                 | Architectures   | Lifecycle
+--------------------------------------|-------------------------|-----------------|----------
+[Windows Client][Windows-client]      | 7 SP1, 8.1              | x64, x86        | [Windows][Windows-lifecycle]
+[Windows 10 Client][Windows-client]   | Version 1607+           | x64, x86, ARM64 | [Windows][Windows-lifecycle]
+[Windows Server][Windows-Server]      | 2012 R2+                | x64, x86        | [Windows Server][Windows-Server-lifecycle]
+[Windows Server Core][Windows-Server] | 2012 R2+                | x64, x86        | [Windows Server][Windows-Server-lifecycle]
+[Nano Server][Nano-Server]            | Version 1809+           | x64             | [Windows Server][Windows-Server-lifecycle]
 
-See the [Windows Lifecycle Fact Sheet](https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet) for details regarding each Windows release lifecycle.
+[Windows-client]: https://www.microsoft.com/windows/
+[Windows-lifecycle]: https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet
+[win-client-docker]: https://hub.docker.com/_/microsoft-windows
+[Windows-Server-lifecycle]: https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info
+[Nano-Server]: https://docs.microsoft.com/windows-server/get-started/getting-started-with-nano-server
+[Windows-Server]: https://docs.microsoft.com/windows-server/
 
-## macOS
+## Linux
 
-OS                            | Version                       | Architectures  | Notes
-------------------------------|-------------------------------|----------------|-----
-Mac OS X                      | 10.13+                        | x64            | [Apple Support Sitemap](https://support.apple.com/sitemap) <br> [Apple Security Updates](https://support.apple.com/en-us/HT201222)
+OS                                    | Version               | Architectures     | Lifecycle
+--------------------------------------|-----------------------|-------------------|----------
+[Alpine Linux][Alpine]                | 3.11+                 | x64, ARM64        | [Alpine][Alpine-lifecycle]
+[CentOS][CentOS]                      | 7+                    | x64               | [CentOS][CentOS-lifecycle]
+[Debian][Debian]                      | 9+                    | x64, ARM32, ARM64 | [Debian][Debian-lifecycle]
```

Comment:
I see that arm64 is supported for Debian but not CentOS. Is this about availability via package managers? Or are "portable" linux-arm64 binaries just not supported for CentOS?

Reply:
I think the portable binaries will work. @richlander - can you confirm?

Reply:
I suspect they will work. It's more a question of what we're testing. As you know, we're much more limited with testing on ARM64.

I see two choices that would enable us to list CentOS:

1. Only list configurations that we actively test, and add CentOS to that mix.
2. List configurations that we expect will work, but with an extra marking to clarify expectations.

This will get one-step more complicated when we add container support to the list. I don't think we'll be able to use choice 1 with everything for both raw-metal and containers. That leads me to believe that we should start with considering choice 2 as a general approach, and then decide which configurations we want to provide better support for with choice 1. Make sense?

FYI @omajid: this is a two-step process. We wanted to update this page to make it correct for .NET 5 (that's what we're doing here), and then the next step is to add Container and Package manager support. @leecow can share our draft of that if you want to take a look / give feedback.